docs/src/main/tut/datatypes/either.md

```diff
@@ -212,132 +212,6 @@ magic("123") match {
 }
 ```
 
-## Either in the small, Either in the large
-Once you start using `Either` for all your error-handling, you may quickly run into an issue where
```

Comment:
@adelbertc The idea is that all the stuff about the left sides needing to be the same will be a lot less relevant (or not at all relevant, in the form here) once the syntax `flatMap` for `Either` behaves the same as the real `flatMap` method on `Either` in 2.12.